README.md

```diff
@@ -10,6 +10,76 @@ It can be used by installing this package & setting the `extends` property in yo
 }
 ```
 
+Alternative yml config  `.eslintrc.yml`:
```

Comment:
Aaah. Didn't know it worked too. I will update README.md with recommendation to use `.eslintrc.js` instead.